Networking/Networking/Model/Refund/OrderItemRefund.swift

```diff
@@ -0,0 +1,137 @@
+import Foundation
+
+
+/// Represents an Order Item to be Refunded
+///
+public struct OrderItemRefund: Codable {
+    public let itemID: Int
+    public let name: String
+    public let productID: Int
+    public let variationID: Int
+    public let quantity: NSDecimalNumber
+    public let price: NSDecimalNumber
```

Comment:
I think `NSDecimalNumber` is used mostly in generated Core Data subclasses. In the Networking layer, could these `NSDecimalNumber` properties be `Int` or `Decimal` so that the conversion is not needed in this layer?

Reply:
Added a comment to explain why it is needed for `price`. I will change quantity to be a `Decimal`.